.github/workflows/test-execute.yml

```diff
@@ -0,0 +1,47 @@
+name: TestExecute
+
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Test Execute Image
+    steps:
+      -
+        name: Checkout repo
+        uses: actions/checkout@v2
+      -
+        name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      -
+        name: Build execute image
+        uses: docker/build-push-action@v2
+        with:
+          file: ./dockerfiles/execute/Dockerfile
+          push: false
+          tags: makerdao/execute:latest
```

Comment:
I get that this is configured to `push: false`, but I'd still prefer to use a different tag so that if we mess that up we don't push/override a `latest` image. Maybe something like `makerdao/execute:gh_actions_test_version`?

Reply:
👍  - I meant to change the name in the first place.